v2/RNHealthKitCore/ReactNativeBridge/RNHealthKitWrapper.swift

```diff
@@ -0,0 +1,121 @@
+import Foundation
+import HealthKit
+
+@objc(RNHealthKitWrapper)
+class RNHealthKitWrapper: NSObject {
+    var core: HealthKitCore?
+
+    @objc
+    func initHealthKit(_ read: Array<String>, write: Array<String>, resolve: @escaping RCTPromiseResolveBlock, reject: @escaping RCTPromiseRejectBlock) {
```

Comment:
nitpick:


nitpick: 
```suggestion
    func initHealthKit(_ read: [String], write: [String], resolve: @escaping RCTPromiseResolveBlock, reject: @escaping RCTPromiseRejectBlock) {
```